README.md

```diff
@@ -117,4 +117,198 @@ containers:
+    message: Spec sync OK
+    reason: SyncOK
+    status: "True"
+    type: SpecSynced
+  instances:
+  - instanceID: 608df625aa94426b4169999
+    instanceInfo:
+      dbaas-cluster1.a00aa.Srv: mongodb+srv://dbaas-cluster1.a00aa.mongodb.net
+      instanceSizeName: M0
+      projectID: 608df5e652e1944293e11111
+      projectName: Project 1
+      providerName: TENANT
+      regionName: US_EAST_1
+    name: DBaaS-Cluster1
+  - instanceID: 60807282b4ab8d3b3c123456
+    instanceInfo:
+       dbaas-cluster1.a00aa.Srv: mongodb+srv://test123.edbca.mongodb.net
+      instanceSizeName: M10
+      projectID: 6065e15b16c0731bf3a12333
+      projectName: Project 2
+      providerName: AWS
+      regionName: US_EAST_1
+    name: test
+  - instanceID: 12345ffbc9a90e310e642482
+    instanceInfo:
+      dbaas-cluster1.a00aa.Srv: mongodb+srv://testcluster1.edbca.mongodb.net
+      instanceSizeName: M0
+      projectID: 6065e15b16c0731bf3a12333
+      projectName: Project 2
+      providerName: TENANT
+      regionName: US_EAST_1
+    name: DBCreatedInAtalas
+  - instanceID: 60b7a72f4877d05881234567
+    instanceInfo:
+      dbaas-cluster1.a00aa.Srv: mongodb+srv://test.abcd9.mongodb.net
+      instanceSizeName: M10
+      projectID: 60b798fea37f9f09acc12345
+      projectName: mytest
+      providerName: AWS
+      regionName: US_EAST_1
+    name: test
+```
+**3.** Check MongoDBAtlasConnection Custom Resource
+
+Now the application developer can create a [DBaaSConnection](https://github.com/RHEcosystemAppEng/dbaas-operator/blob/main/config/crd/bases/dbaas.redhat.com_dbaasconnections.yaml) CR for connection to the MongoDB database instance found, the DBaaS Operator automatically creates a MongoDBAtlasConnection CR. The Atlas Operator creates a database user in Atlas for the cluster with the default database `admin`. The Atlas Operator stores the db user credentials in a kubernetes secret, and the remaining connection information in a configmap, and then updates the MongoDBAtlasConnection CR status.
+
+Here is an example of MongoDBAtlasConnection CR.
+```
+apiVersion: dbaas.redhat.com/v1alpha1
+kind: MongoDBAtlasConnection
+metadata:
+  name: test-dbaas-connection
+  namespace: test-namespace
+  ownerReferences:
+  - apiVersion: dbaas.redhat.com/v1alpha1
+    blockOwnerDeletion: true
+    controller: true
+    kind: DBaaSConnection
+    name: test-dbaas-connection
+    uid: 77193619-6ab1-43c9-acf2-a40c2cfe7703
+spec:
+  instanceID: 12345ffbc9a90e310e642482
+  inventoryRef:
+    name: dbaas-mytest
+    namespace: openshift-operators
+status:
+  conditions:
+  - lastTransitionTime: "2021-08-18T20:07:51Z"
+    message: ""
+    reason: Ready
+    status: "True"
+    type: ReadyForBinding
+  connectionInfoRef:
+    name: atlas-connection-cm-knp9z
+  credentialsRef:
+    name: atlas-db-user-5pc8b
+```
+The corresponding generated secret:
+```
+apiVersion: v1
+data:
+  password: cGFzczEyM3dAcmQ=
+  username: ZGJVc2VyXzEwMQ==
```

Comment:
I still feel like these should be redacted here, even though we're redacting the URL where they could be used... if that password value, which is a very easy decode,is used as a default on user creation or something this is not good to push out.

Reply:
These are already redacted with dummy values: dbUser_101/pass123w@rd.

Reply:
ok thanks, I decoded the values and the pw seemed like a legit one, so just wanted to make sure.